<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>451</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Крен</string>
     </attribute>
     <widget class="QCustomPlot" name="rollPlot" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>441</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Тангаж</string>
     </attribute>
     <widget class="QCustomPlot" name="pitchPlot" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>441</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Рыскание</string>
     </attribute>
     <widget class="QCustomPlot" name="yawPlot" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>441</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>451</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>X</string>
     </attribute>
     <widget class="QCustomPlot" name="xPlot" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>441</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Y</string>
     </attribute>
     <widget class="QCustomPlot" name="yPlot" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>441</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Z</string>
     </attribute>
     <widget class="QCustomPlot" name="zPlot" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>441</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>451</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Мотор 1</string>
     </attribute>
     <widget class="QCustomPlot" name="rotor1Plot" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>441</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Мотор 2</string>
     </attribute>
     <widget class="QCustomPlot" name="rotor2Plot" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>441</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>Мотор 3</string>
     </attribute>
     <widget class="QCustomPlot" name="rotor3Plot" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>441</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_11">
     <attribute name="title">
      <string>Мотор 4</string>
     </attribute>
     <widget class="QCustomPlot" name="rotor4Plot" native="true">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>441</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>451</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_10">
     <attribute name="title">
      <string>Управление креном</string>
     </attribute>
     <widget class="QCustomPlot" name="controlRollPlot" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>441</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_12">
     <attribute name="title">
      <string>Управление тангажом</string>
     </attribute>
     <widget class="QCustomPlot" name="controlPitchPlot" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>441</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_13">
     <attribute name="title">
      <string>Управление рысканьем</string>
     </attribute>
     <widget class="QCustomPlot" name="controlYawPlot" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>441</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>160</y>
      <width>311</width>
      <height>556</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="13" column="1">
      <widget class="QLineEdit" name="yawLineEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Начальная позиция аппарата</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Начальное положение аппарата</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>OZ (м)</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Тангаж (градусы)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Шаг моделирования</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="timeStepSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
       <property name="value">
        <double>0.025000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>OX (м)</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Тангаж (градусы)</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Рысканье (градусы)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="16" column="1">
      <widget class="QLineEdit" name="desiredRollLineEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="positionZLineEdit">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="positionXLineEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="positionYLineEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Рысканье (градусы)</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0" colspan="2">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="endTimeSpinBox">
       <property name="singleStep">
        <double>0.000100000000000</double>
       </property>
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Крен (градусы)</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QLineEdit" name="desiredYawLineEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QLineEdit" name="desiredPitchLineEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkBox_5">
       <property name="text">
        <string>Включить</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Крен (градусы)</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0" colspan="2">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Требуемое положение</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="rollLineEdit">
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>OY (м)</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Ветер</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="pitchLineEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Время моделирования</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0" colspan="2">
      <widget class="QPushButton" name="startSimulationButton">
       <property name="text">
        <string>Начать</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Параметры симуляции</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>0</y>
      <width>311</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>Параметры ПИД-регуляторов </string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="rollKiSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="singleStep">
        <double>0.000010000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Крен</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Тангаж</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>Kd</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="pitchKpSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="singleStep">
        <double>0.000010000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QDoubleSpinBox" name="pitchKiSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="singleStep">
        <double>0.000010000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="rollKpSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="singleStep">
        <double>0.000010000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="yawKpSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="singleStep">
        <double>0.000010000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Kp</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>Рыскание</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QDoubleSpinBox" name="yawKdSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="singleStep">
        <double>0.000010000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QDoubleSpinBox" name="yawKiSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="singleStep">
        <double>0.000010000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDoubleSpinBox" name="rollKdSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="singleStep">
        <double>0.000010000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QDoubleSpinBox" name="pitchKdSpinBox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="singleStep">
        <double>0.000010000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>Ki</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="4">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>Применить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1229</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
